package.json

```diff
@@ -2,13 +2,15 @@
   "name": "capitaljs",
   "version": "0.0.1",
   "description": "Financial formulas in JavaScript",
-  "main": "src/index.js",
+  "main": "lib/index.js",
   "private": true,
   "scripts": {
     "test": "tape test/*.js | faucet",
     "bundle": "browserify src/index.js --standalone capitaljs -t es6ify | derequire > capital.js",
     "compress": "uglifyjs capital.js -o capital.min.js",
     "build": "npm run bundle && npm run compress",
+    "babel-build": "cd src; babel ./**.js -d ../lib && babel ./**/*.js -d ../lib",
```

Comment:
Have you tried just `babel src -d lib`? I think it does the same thing and is a lot cleaner.